projects/1.18/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -1 +1,555 @@
+  "gui.seismicscanner.ycoord" : "Y坐标：%s",
+  "gui.seismicscanner.zcoord" : "Z坐标：%s",
+  "gui.seismicscanner.xcoordna" : "X坐标：未知",
+  "gui.seismicscanner.ycoordna" : "Y坐标：未知",
+  "gui.seismicscanner.zcoordna" : "Z坐标：未知",
+  "gui.seismicscanner.material" : "方块材质：",
+  "gui.genericcharger.chargeperc" : "充电进度：%s",
+  "gui.genericcharger.chargecapable" : "充电效率：%s",
+  "gui.seismicrelay.dataheader" : "标记",
+  "gui.seismicrelay.posnotfound" : "%s：未找到",
+  "gui.seismicrelay.posfound" : "%1$s：%2$s",
+  "gui.seismicrelay.posoptional" : "%s：找到多个",
+
+  "gui.quarry.voiditems" : "采石场",
+  "gui.quarry.needvoidcard" : "需要卡片",
+  "gui.quarry.status" : "状态：",
+
+
+  "gui.quarry.setup" : "部署中……",
+  "gui.quarry.mining" : "挖掘中……",
+  "gui.quarry.finished" : "完成！",
+  "gui.quarry.error" : "错误",
+  "gui.quarry.running" : "运行中……",
+  "gui.quarry.needspower" : "电力不足！",
+
+
+  "gui.quarry.stats" : "数据：",
+  "gui.quarry.fortune" : "时运：%s",
+  "gui.quarry.silktouch" : "精准采集：%s",
+  "gui.quarry.unbreaking" : "耐久：%s",
+  "itemGroup.itemgroupelectrodynamics" : "电动力学",
+  "tooltip.itemwire.resistance" : "电阻：%s",
+  "tooltip.itemwire.maxamps" : "最大电流：%s",
+  "tooltip.itemwire.info.insulated" : "电压≤240.0V时不发生触电",
+  "tooltip.itemwire.info.highlyinsulated" : "电压≤960.0V时不会触电",
+  "tooltip.itemwire.info.ceramic" : "电压≤480.0V时不会触电",
+  "tooltip.itemwire.info.uninsulated" : "会发生触电",
+  "tooltip.itemwire.info.logistical" : "通电时发出红石信号",
+  "tooltip.info.capacityupgrade" : "将电能储量和传输速率提升至原来的%s倍",
+  "tooltip.info.capacityupgradevoltage" : "将机器最大通电电压提高到最大%s",
+  "tooltip.info.speedupgrade" : "将处理速度提升到原来的%s倍",
+  "tooltip.info.itemoutputupgrade" : "将物品弹出至附近的容器中",
+  "tooltip.info.iteminputupgrade" : "从附近的容器中抽取物品",
+  "tooltip.info.dirlist" : "当前方向列表：",
+  "tooltip.info.nodirs" : "Shift+右击以添加一个方向",
+  "tooltip.info.cleardirs" : "Shift+单击以清除选中方向",
+  "tooltip.info.insmartmode" : "智能模式",
+  "tooltip.info.togglesmart" : "右击以打开智能模式",
+  "tooltip.info.guidebookuse" : "Shift+右击以打开指南",
+  "tooltip.info.guidebooktemp" : "敬请期待",
+  "tooltip.info.concretejoke" : "万物之始",
+  "tooltip.info.xpstored" : "经验储量：%s",
+  "tooltip.info.xpusage" : "Shift+右击以取出经验",
+  "tooltip.info.range" : "将半径扩大1格",
+  "tooltip.info.broken" : "机器已损坏",
+  "tooltip.transformer.energyloss" : "电能转换效率：99.25%",
+  "tooltip.machine.voltage.120" : "电压：120 V",
+  "tooltip.machine.voltage.240" : "电压：240 V",
+  "tooltip.machine.voltage.480" : "电压：480 V",
+  "tooltip.machine.voltage.960" : "电压：960 V",
+  "tooltip.item.electric.info" : "电量：%s",
+  "tooltip.item.electric.voltage" : "输入/输出电压：%s",
+
+  "tooltip.electrodynamics.railguntemp" : "温度：%s",
+  "tooltip.electrodynamics.railgunmaxtemp" : "最大温度：%s",
+  "tooltip.electrodynamics.railgunoverheat" : "警告：过热！",
+  "tooltip.electrodynamics.ceramicplatecount" : "剩余陶瓷板数：%s",
+  "tooltip.tanksteel.capacity" : "容量：8000 mB",
+  "tooltip.tankreinforced.capacity" : "容量：32000 mB",
+  "tooltip.tankhsla.capacity" : "容量：120000 mB",
+  "tooltip.creativepowersource.joke" : "\"UNLIMITEDPOWER!\"-希夫·帕尔帕廷",
```

```suggestion
  "tooltip.creativepowersource.joke" : "\"无尽能源！\"-星际迷航中的巴克·罗杰斯",
```
 原文为"\"Unlimited Power\" - Buck Rogers from Star Trek",还是说另有典故？

---

先去查查这个梗到底来源于哪再说…另外不排除这是一种外文版的“指虎为刀”

---

感觉星际迷航中的这几个字有点小出戏。

---

[“UNLIMITED POWER!” - YouTube](https://www.youtube.com/watch?v=SWwFogRQVnk)